packages/devtools_app/lib/src/memory/memory_controller.dart

```diff
@@ -431,16 +463,57 @@ class LibraryCollection {
       findOtherLibrary(displayClasses[classId]) != null;
 }
 
+/// Prepare data to plot in MPChart.
+class MPChartData {
+  /// Datapoint entry for each used heap value.
```

Comment:
nit for here and the following two comments - r/entry/entries

Reply:
Done.